### PR TITLE
Add A6 family code for DS2438 1-Wire sensor

### PR DESCRIPTION
**Pull request by definitio**
## Proposed change

Add A6 family code for DS2438 1-Wire sensor

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/112844
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Review by epenet — COMMENTED** [review]
It feels very weird to have two separate prefixes for the same device.
Are you 100% sure that this is a DS2438?
Do you have a reference to dallas indicating this?

If it's a "similar" device or a new generation, then please point out the source.

File: source/_integrations/onewire.markdown [hunk 1]
```diff
@@ -52,6 +52,7 @@ Each 1-wire component data sheet describes the different properties the componen
 | 3B     | [DS1825](https://datasheets.maximintegrated.com/en/ds/DS1825.pdf)            | Temperature                     |
 | 42     | [DS28EA00](https://datasheets.maximintegrated.com/en/ds/DS28EA00.pdf)        | Temperature                     |
 | 7E     | [EDS00xx](https://www.embeddeddatasystems.com/assets/images/supportFiles/manuals/EN-UserMan%20%20OW-ENV%20Sensor%20v13.pdf)        | Temperature/Humidity/Barometric pressure/Light <sup>[6](#note_6)</sup>|
+| A6     | [DS2438](https://datasheets.maximintegrated.com/en/ds/DS2438.pdf)            | Temperature, Voltage, Current (pressure when using B1-R1-A, illuminance when using S2-R1-A, humidity when using compatible Honeywell or Humirel sensor) <sup>[2](#note_2)</sup> |
```
Comment (epenet) [inline source/_integrations/onewire.markdown]:
```suggestion
| A6     | Secondary family code for [DS2438](https://datasheets.maximintegrated.com/en/ds/DS2438.pdf)            | Temperature, Voltage, Current (pressure when using B1-R1-A, illuminance when using S2-R1-A, humidity when using compatible Honeywell or Humirel sensor) <sup>[2](#note_2)</sup> |
```

**Review by epenet — CHANGES_REQUESTED** [review]

**Review by epenet — APPROVED** [review]
LGTM

**Review by c0ffeeca7 — APPROVED** [review]
Thank you, @definitio and @epenet  :+1:

**Pull request merged**